src/KeyManager/MemoryKey/index.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai'
+import MemoryKey from '.'
+import { InvalidMnemonic } from '../errors'
+import { AddressType } from '../../Wallet'
+import { getBindingsForEnvironment } from '../../lib/bindings'
+import { generateTestTransaction } from '../../utils/test/test_transaction'
+import { generateMnemonic } from '../../utils'
+const { BlockchainSettings } = getBindingsForEnvironment()
+
+describe('MemoryKey', () => {
+  it('throws if the mnemonic passed is invalid', () => {
+    const invalidMnemonic = 'xxxx'
+    expect(() => MemoryKey({ mnemonic: invalidMnemonic, password: 'xx' })).to.throw(InvalidMnemonic)
+  })
+
+  describe('publicAccount', () => {
+    it('exposes a Bip44 public account', () => {
+      const mnemonic = generateMnemonic()
+      const keypair = MemoryKey({ mnemonic, password: 'securepassword' })

```

Comment:
Let's be consistent with the terminology
```suggestion
      const keyManager = MemoryKeyManager({ mnemonic, password: 'securepassword' })
```